includes/class-wc-stripe-checkout-sessions-controller.php

```
@@ -35,10 +35,12 @@ public function create_checkout_session(): void {
 
 			$payment_method_type     = isset( $_POST['payment_method_type'] ) ? wc_clean( wp_unslash( $_POST['payment_method_type'] ) ) : '';
 			$enabled_payment_methods = $payment_method_type ? [ $payment_method_type ] : [];
+			$wc_customer             = WC()->customer;
```

PHPMD (2.15.0)

[error] The variable $wc_customer is not named in camelCase. (undefined)

(CamelCaseVariableName)